.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 63401
```

Comment:
This file should not be part of the git repository.

Reply:
🙈